api/service/metrics/client.go

```diff
@@ -0,0 +1,50 @@
+package metrics
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"google.golang.org/grpc"
+)
+
+// Constants for client service.
+const (
+	Port = "29000"
```

Comment:
don't use a fixed port.  Use some offset, as we will need to support multiple node programs running on the same host.